frontend/homework/9/hello-world/src/components/UserUI.tsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import {Hobbies} from "./Hobbies";
+import {Info} from "./Info";
+import {Skills} from "./Skills";
+import { fetchPayload, IPayload } from './api';
+import './main.css'
+
+export default function UserUI() {
+    const [lists, setLists] = useState<IPayload | null>(null);
+
+    useEffect(() => {
+        async function fetchData() {
+            const data = await fetchPayload();
+            setLists(data);
+        }
+        fetchData();
+    }, []);
+
+
+    if (lists === null) {
+        return <div>Loading...</div>;
+    }
```

Comment:
return (  
condition ? <>...<> : <>...<>
)

Conditional rendering can also be done inline